Duplicati/Library/Utility/Utility.cs

```diff
@@ -18,6 +18,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Threading;

```

Comment:
It looks like this `using` directive already exists on line 28 below.

Reply:
Thanks for pointing this out - I'm not sure how this happened. Maybe Visual Studio automatically added it?

At any rate, it's now cleaned up.